recipes/ner_pre_annotations_openai.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    - for location named entity
</pre><p>remove -for ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>